scripts/generate_docs.sc

```diff
@@ -0,0 +1,101 @@
+#!/usr/bin/env amm
+
+import java.time.Instant
+import ammonite.ops._
+import ammonite.ops.ImplicitWd._
+
+/**
+  * Docs generation process:
+  *
+  * 1. copy docs/docs and docs/_layouts/docs.html part from 1.3, 1.4 and 1.5 branches to the temp dirs
+  * 2. copy the latest release (1.5_ branch doc to the temp dir
+  * 3. generate the latest release docs
+  * 4. replace docs/docs with docs/docs from 1.3, 1.4, 1.5 and generate docs for the _sites/docs_version
+  */
+
+def makeTmpDir(): Path = {
+  val timestamp = Instant.now().toString.replace(':', '-')
+  val path = root/"tmp"/s"marathon-docs-build-$timestamp"
+  mkdir! path
+  path
+}
+
+val latestReleaseVersion = "1.5"
```

Comment:
can we somehow make this not manual? I mean now we have to think that with every release, we must update this script...

Reply:
It's just a name of the release branch. I don't think it will be changed too often

Reply:
I agree that this shouldn't be hardcoded. It should be rather simple to compute the latest version.

However I think for the first step it's ok to have it manual. Let's iterate from there and improve in the next step.

What I'd suggest to get rid of the hardcoded values and prevent us from publishing docs based on unreleased commits:
1. list all `vX.Y.Z` **tags**, sort them and take the 3 latest _minor_ tags (currently this should result in 1.3.13, 1.4.11, 1.5.7). We don't plan so bump the major version any time soon.
2. Add `master`. We should always generate docs for master.
3. Generate docs for the resulting list of ("1.3.13", "1.4.11", "1.5.7", "master").

Note: It makes sense to me that the docs are contained in _major.minor_  folders. It would be great to add information about which patch release is the documented one (e.g. "1.5.7"), but maintain those docs in a "1.5" folder.

Reply:
`SemVer` class from this PR will be helpful once merged:
https://github.com/mesosphere/marathon/pull/5971/files#diff-06a37bc5ba23ab5621562efe4a70a0e1R141